### DISCUSSION TITLE
Navigation overview remaining route

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#94] [Q&A] [answered]
Concerning the overview button which now shows the complete route: this now switches between full overview and actual view. So far so fine. I would like to propose an enhanced function.

The enhancement could contain the proposed "partly overview" which could switch between different zoom modes (% of the complete route) starting from the actual gps position on every tap, for example
1st tap: next 20% (with fallback to actual view after timer countdown)
2nd tap (within the timer): next 40% (with fallback to actual view after timer countdown)
3rd tap (within the timer): next 60% (with fallback to actual view after timer countdown)
3th tap (within the timer): 100% (with fallback to actual view after timer countdown)

For the free compass tap we can think of another fancy idea for it  later.

**Comment by devemux86** [comment]
@DolomitesTom thanks for the interesting idea!

We could also improve the current overview, displaying not the whole route,
but the untraveled route in front of user's location and up to the destination?

**Reply by DolomitesTom** [reply]
yes that's in principle what I meant, showing the untraveled route in front of user's location in 3-4 different zoom modes, last could be up to the destination.

**Comment by MotoUKRider** [comment]
To throw another idea into this thread, how about also showing the route between each Via Point? So rather than a %, it would show the "stages" of your trip.

**Reply by DolomitesTom** [reply]
If you mean waypoints, then yes, also good idea, but then it can be more than 3 different zoom modes. Some routes have more via points. Shaping points should ignored in any case, since we need more than before to use the curvy roads. By the way, can somebody ask the Brouter developer to create a curvy route option :-) ? It's not really a must, but why not ?

**Comment by devemux86** [comment]
"Route overview" navigation action (tap direction icon at top left)
will display the overview of the **remaining** route in next update.

**Reply by devemux86** [reply]
> "Route overview" navigation action (tap direction icon at top left)
will display the overview of the remaining route in next update.

Available for testing in Cruiser 5.0.1-b1:
- https://github.com/devemux86/cruiser/discussions/641#discussioncomment-10956572

**Comment by devemux86** [comment, marked as answer]
- Navigation overview remaining route is available in https://github.com/devemux86/cruiser/discussions/641
